doc/user-guide/tcs-user-guide-generic.rst

```diff
@@ -342,38 +350,76 @@ In :numref:`fig-offset-xy-example` we show an example of the result of using :py
 In addition to the different methods shown above, users also have control of how the offsets are treated by the pointing component.
 For each command, there are two additional control flags the user can rely on to change the underlying behavior.
 
-.. _user-guide-generic-telescope-control-operations-relative-vs-absolute-offsets:
+.. _user-guide-generic-telescope-control-operations-relative-vs-non-relative-offsets:
 
-Relative vs Absolute offsets
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Relative vs Non-Relative Offsets
```

Comment:
So I'm not a big fan of "Non-Relative"  ... but i can't think of a better suggestion so I guess it stays!

Reply:
I can definitely confirm the docs didn't get rebuilt as I still see "Relative vs Absolute" in the rendered text.